climada/hazard/tc_tracks.py

```diff
@@ -1249,12 +1258,70 @@ def from_netcdf(cls, folder_name):
                                "whole life time.")
                 basin = track.basin
                 del track.attrs['basin']
-                track['basin'] = ("time", np.full(track.time.size, basin))
+                track['basin'] = ("time", np.full(track.time.size, basin, dtype="<U2"))
             data.append(track)
         tr = cls()
         tr.data = data
         return tr
 
+    def write_hdf(self, file_name, complevel=4):
+        """Write TC tracks in HDF5 format.
+
+        Parameters
+        ----------
+        file_name: str or Path
+            Path to a new HDF5 file. If it exists already, the file is overwritten.
```

Comment:
In the rest of CLIMADA it is usual to automatically overwrite. Do you think this is a good idea? This is not necessarily to make a change here right now, unless it is decided that overwriting should not be the default behavior.

Reply:
Yes, this is absolutely fine. It's the same for functions like `pandas.DataFrame.to_csv` or `xarray.Dataset.to_netcdf` or `pickle.dump`.

Reply:
Sure, some functions do it this, some differently :). I personally do not like automatic overwrite, but if you prefer it this way I have no objections.